{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -48,7 +48,14 @@ repos:
     hooks:
       - id: djlint-reformat-django
       - id: djlint-django
-
+{%- if cookiecutter.use_heroku == "y" %}
```

Comment:
This file doesn't contain any jinja tags right now in order to make `pre-commit autoupdate` work. Unwanted blocks are removed in post-generation hooks.

Unless pre-commit handles that now, this will break.

Reply:
https://github.com/cookiecutter/cookiecutter-django/blob/fb27881b33aabe4273dab9703df0f420db67c70d/hooks/post_gen_project.py#L220

Reply:
Ah yes, that is much cleaner. 

Planning to do the same for dependencies in pyproject.toml so we can have a single lockfile for the complete project, but not in this PR ;)